includes/class-wc-stripe-intent-controller.php

```
@@ -1394,6 +1394,10 @@ public function confirm_change_payment_from_setup_intent_ajax() {
 			$gateway->set_customer_id_for_subscription( $subscription, $customer->get_id() );
 			$gateway->set_payment_method_id_for_subscription( $subscription, $token->get_token() );
 
+			// Mirror the no-3DS change-payment branch: associate the new WC token with
+			// the subscription so My Account → Subscriptions reflects the new card.
+			WC_Stripe_Express_Checkout_Helper::replace_subscription_payment_token( $subscription, $token->get_token() );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Express_Checkout_Helper' in method 'confirm_change_payment_from_setup_intent_ajax'. (undefined)

(StaticAccess)